packages/atomic/src/components/facets/facet-segmented-value/facet-segmented-value.pcss

```diff
@@ -0,0 +1,24 @@
+.box-container {
+  @apply flex;
+}
+
+.box-container li:first-child .value-box {
+  border-radius: var(--atomic-border-radius) 0px 0px var(--atomic-border-radius);
```

Comment:
This should be do-able with tailwind shorthands. For instance, the `rounded` class applies `border-radius: var(--atomic-border-radius)`. The reason it applies our variable is because it corresponds with "DEFAULT" in `tailwind.config.js`.